common/v2/components/Tooltip.tsx

```diff
@@ -3,14 +3,33 @@ import React from 'react';
 import { Tooltip as UITooltip } from '@mycrypto/ui';
 
 import questionSVG from 'assets/images/icn-question.svg';
+import informationalSVG from 'assets/images/icn-info-blue.svg';
+
+export enum IconID {
+  question = 'question',
```

Comment:
Do they need to be strings?

Reply:
No, but the strings are easier to read imo.